iDiff/utils/diff_utils.go

```diff
@@ -0,0 +1,60 @@
+package utils
+
+import (
+	"github.com/pmezard/go-difflib/difflib"
+)
+
+// Modification of difflib's unified differ
+func GetAdditions(a, b []string) []string {
+	matcher := difflib.NewMatcher(a, b)
+	differences := matcher.GetGroupedOpCodes(0)
+
+	var adds []string
+	for _, g := range differences {
```

Comment:
Can you make c and g more descriptive names?  I'm not really sure what they are because I don't know what GetGroupedOpCodes returns without looking up the library...